ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/AddExtensionAttributeFix.fs

```diff
@@ -0,0 +1,51 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util.FSharpAttributesUtil
+
+type AddExtensionAttributeFix(warning: ExtensionMemberInNonExtensionTypeWarning) =
+    inherit FSharpQuickFixBase()
+
+    let [<Literal>] extensionNamespaceName = "System.Runtime.CompilerServices"
+    let [<Literal>] attributeName = "Extension"
+    let declaration =
+        let attributeHolder: IFSharpTypeMemberDeclaration =
```

Comment:
Please use `owner` word to match R# Psi naming (e.g. `attributesOwner`, `attributesOwnerDecl`, or `attributesOwnerDeclaration`).